docs/sampling.asciidoc

```diff
@@ -1,27 +1,77 @@
 [[sampling]]
 === Transaction sampling
 
-Elastic APM supports head-based, probability sampling.
-_Head-based_ means the sampling decision for each trace is made when that trace is initiated.
-_Probability sampling_ means that each trace has a defined and equal probability of being sampled.
+Distributed tracing can generate a substantial amount of data.
+More data can mean higher costs and more noise to sift through.
+Sampling aims to lower the amount of data ingested and the effort required to analyze that data --
+all while still making it easy to find anomalous patterns in your applications, detect outages, track errors,
+and lower MTTR.
+
+Elastic APM supports two types of sampling:
+
+* <<head-based-sampling>>
+* <<tail-based-sampling>>
+
+[float]
+[[head-based-sampling]]
+==== Head-based sampling
+
+In head-based sampling, the sampling decision for each trace is made when that trace is initiated.
+Each trace has a defined and equal probability of being sampled.
 
 For example, a sampling value of `.2` indicates a transaction sample rate of `20%`.
 This means that only `20%` of traces will send and retain all of their associated information.
 The remaining traces will drop contextual information to reduce the transfer and storage size of the trace.
```

Comment:
Since 8.0, we drop non-sampled transactions (except for RUM) in APM Server. Should we update this?

Reply:
Ah, that's right. I'll update.

Reply:
Should we remove "The remaining traces will drop contextual information to reduce the transfer and storage size of the trace."? Some agents will start automatically dropping non-sampled transactions agent-side, when they detect a new enough version of APM Server. In the future this should be the default.

Maybe say they'll be dropped (without specifying where) and link to `Sampled data and visualizations`?